streaming/src/main/scala/io/gearpump/streaming/task/TaskActor.scala

```diff
@@ -242,6 +243,16 @@ class TaskActor(
       receiveMessage(inputMessage, sender)
     case upstream@ UpstreamMinClock(upstreamClock) =>
       this.upstreamMinClock = upstreamClock
+      val curMin = minClockAtCurrentTask
+      if (curMin == curTaskMinClock) {
```

Comment:
It is confusing to see both minClockAtCurrentTask and curTaskMinClock.

what is the difference?